sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/sin-validation-integration/sin-validation-process-response-integration.scheduler.ts

```diff
@@ -15,57 +17,35 @@ export class SINValidationResponseIntegrationScheduler extends BaseScheduler<voi
     schedulerQueue: Queue<void>,
     queueService: QueueService,
     private readonly sinValidationProcessingService: SINValidationProcessingService,
-    private readonly systemUsersService: SystemUsersService,
   ) {
     super(schedulerQueue, queueService);
   }
 
-  /**
-   * To be removed once the method {@link process} is implemented.
-   * This method "hides" the {@link Process} decorator from the base class.
-   */
-  async processQueue(): Promise<string | string[]> {
-    throw new Error("Method not implemented.");
-  }
-
-  /**
-   * When implemented in a derived class, process the queue job.
-   * To be implemented.
-   */
-  protected async process(): Promise<string | string[]> {
-    throw new Error("Method not implemented.");
-  }
-
   /**
    * Download all SIN validation files from ESDC response folder on SFTP and process them all.
-   * @params job job details.
+   * @param _job process job.
+   * @param processSummary process summary for logging.
    * @returns summary with what was processed and the list of all errors, if any.
    */
-  @Process()
-  async processSINValidationResponse(
-    job: Job<void>,
-  ): Promise<ProcessResponseQueue[]> {
-    const summary = new QueueProcessSummary({
-      appLogger: this.logger,
-      jobLogger: job,
-    });
-    await summary.info(
-      `Processing SIN validation integration job ${job.id} of type ${job.name}.`,
-    );
-    await summary.info("Processing ESDC SIN validation response files.");
-    const auditUser = this.systemUsersService.systemUser;
-    const results = await this.sinValidationProcessingService.processResponses(
-      auditUser.id,
-    );
-    await summary.info("ESDC SIN validation response files processed.");
```

Comment:
Can we have a `processSummary.info` with similar context ?

Reply:
I am not sure about adding it. I see the point but the `BaseQueue` is already adding the below pair of logs to the `ProcessSummary` when the job starts and when it finishes
```
Processing queue ${job.queue.name}, job ID ${job.id}.
```
```
${job.queue.name}, job ID ${job.id}, executed.
```
_Note_: The base queue was wrongly adding the first log using the `logger` and the second one using the `ProcessSummary` and I just changed it in the last commit. Please take a look.

We need to decide if we need this "friendly" message at the processor level or the messages from the `BaseQueue` would be enough. Please lets have a quick chat and decide one way or another. I can go back and review the refactored schedulers.

Reply:
As we talked, removing the friendly messages from the processor since the `BaseQueue` has some similar to indicate the start and end of the processing.